deploy/charts/harvester/templates/rbac.yaml

```diff
@@ -47,6 +47,14 @@ rules:
       - network-attachment-definitions
     verbs:
       - '*'
+  - apiGroups:
+      - snapshot.storage.k8s.io
+    resources:
+      - volumesnapshots
+      - volumesnapshotcontents
+      - volumesnapshotclasses
```

Comment:
I believe the `volumesnapshotclasses` should only be managed by the cluster admin, and it is a cluster scope CRD